mods/ts/rules/defaults.yaml

```diff
@@ -542,8 +542,14 @@
 	Voiced:
 		VoiceSet: Infantry
 	Targetable:
-		RequiresCondition: !inside-tunnel
 		TargetTypes: Ground, Infantry
+		RequiresCondition: !inside-tunnel
```

Comment:
Style nit: `RequiresCondition` should be the first field on a definition, so keep this where it was.